pkg/core/native/util.go

```diff
@@ -64,6 +70,20 @@ func setUint32WithKey(id int32, dao dao.DAO, key []byte, value uint32) error {
 	return dao.PutStorageItem(id, key, si)
 }
 
+func setBigIntWithKey(id int32, dao dao.DAO, key []byte, value *big.Int) error {
+	si := &state.StorageItem{Value: bigint.ToBytes(value)}
+	return dao.PutStorageItem(id, key, si)
+}
+
+func getBigIntWithKey(id int32, dao dao.DAO, key []byte) *big.Int {
+	si := dao.GetStorageItem(id, key)
+	if si == nil {
+		panic(fmt.Errorf("item with id = %d and key = %s is not initialized", id, hex.EncodeToString(key)))
```

Comment:
This is invoked from public policy methods which are not protected by `execute`. But currently all values are set so maybe it is not a big deal now.

Reply:
It's a program error if those values are not initialised. They should always be set, so I think panic is OK here.